### DISCUSSION TITLE
Dynamic route page fails to load script

### DISCUSSION THREAD

**Discussion by Astraeus-** [vercel/next.js#12214] [Help]
`Loading failed for the <script> with source “http://localhost:3000/_next/static/development/pages/team/Test-Team1.js”.`

I'm navigating to the page at:  `http://localhost:3000/team/Test-Team1` and get the above error. It seems to me that next.js thinks this is a specific route, instead of dynamic one. I do use `getServerSideProps` on these pages, could that be causing this issue?

When I look inside the `_next` folder I do have the js files for the dynamic routes as `[slug].js` and `[slug].js.map`.

Any help or pointers would be appreciated

**Comment by touatiosema** [comment]
I ran into the same issue. i wonder if there is a solution.

**Comment by sa-webb** [comment]
This is happening to me only on a fresh start of my development server, but I have yet to test in production. The page that is causing this error is triggering a conditional redirect that is **NOT** using `getServerSideProps` or `getStaticProps.` Since my use-case allows it I am going to try to prefetch that page and test in production.

**Comment by jeanantunestw** [comment]
Same error here. This is happening  just on my development server, in production is working!

**Comment by sohammondal** [comment]
This issue isn't there in v12 but ran into this when I downgraded it to v11.0.3.

**Comment by jonasguerra** [comment]
I'm having the same problem here, anyone found a solution for this?

**Comment by longlostnick** [comment]
Same. Very confusing.

**Reply by sa-webb** [reply]
I haven't touched Next.js in a while but something that may help is deleting the .next cache folders and restarting your app.
